tests/frontend/test_parameters.py

```diff
@@ -67,7 +69,23 @@ def test_par_is_symbolic(self, r):
         assert par_is_symbolic(p * r)
         assert par_is_symbolic(p / r)
         assert par_is_symbolic(p ** r)
-        assert par_is_symbolic(p / p)  # no simplification
+        assert par_is_symbolic(p - p)  # no simplification
```

Comment:
How come the `p/p` case was dropped? Did it actually provide a simplification? 
If not, why not keep it in tests? If so, why does `p-p` not do the same?

Reply:
`p/p` should maybe not be simplified since `p` might be zero, but `p-p` always simplifies to zero for finite `p`.

Reply:
good point